sauce/features/budget/enter-to-move/index.js

```diff
@@ -0,0 +1,49 @@
+import { Feature } from 'toolkit/core/feature';
+import * as toolkitHelper from 'toolkit/helpers/toolkit';
+
+const MOVE_POPUP =
+  'ynab-u modal-popup modal-budget modal-budget-move-money ember-view modal-overlay active';
+const CATEGORY_DROPDOWN = 'dropdown-container categories-dropdown-container';
+const BUTTON_PRIMARY = 'button button-primary ';
+
+export class EnterToMove extends Feature {
+  modalIsOpen = false;
+
+  constructor() {
+    super();
+    this.onKeyDownHandler = this.onKeyDown.bind(this);
+  }
+
+  shouldInvoke() {
+    return toolkitHelper.getCurrentRouteName().indexOf('budget') !== -1;
+  }
+
+  invoke() {}
+
+  onKeyDown(e) {
+    const keycode = e.keycode || e.which;
+    if (keycode === 13) {
+      const OK = $(
+        '.modal-budget-move-money .modal-actions button:first-child'
+      );
+      OK.click();
+      document.removeEventListener('keydown', this.onKeyDownHandler, false);
+    }
```

Comment:
Need a check for key code of 27 which is the ESC key so the handler can be removed?
Need to check for key code of 18 which is the non-numpad enter key?

Reply:
18 is `Alt`, not numpad enter? and I could add a listener for `esc`, but that's not really in the scope of this plugin. Maybe the full keyboard shortcuts plugin should include this? The handler gets removed when the modal disappears, whether triggered by `Enter` or a mouse click.

Reply:
I forgot the feature is working with a modal that YNAB "controls" which means they handle the ESC key. 

Also true that keycode 18 is the tab key so never mind. Something is niggling me about that keycode but I can't think what it is. So yeah, sucks to have a bad memory.